website/source/docs/providers/aws/r/autoscaling_metrics_collection.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "aws"
+page_title: "AWS: aws_autoscaling_metrics_collection"
+sidebar_current: "docs-aws-resource-autoscaling-metrics-collection"
+description: |-
+  Enables / Disables Autoscaling Group Metrics Collection.
+---
+
+# aws\_autoscaling\_metrics\_collection
+
+Enables / Disables Autoscaling Group Metrics Collection.
+
+~> **NOTE:** You can only enable metrics collection for an Autoscaling Group if `InstanceMonitoring` 
+in it's underlying launch configuration for the group is set to `True`.
```

Comment:
We should provide an example of this in the config below. For instance, the field is `enable_monitoring` in Terraform, correct?